examples/TranslateBinomial2.jl

```diff
@@ -542,6 +560,9 @@ function cellular_associated_primes(I::MPolyIdeal)
 
   #construct the ideal (x_i \mid i \in \Delta^c)
   gi = elem_type(R)[Variables[i] for i = 1:nvars(R) if !(i in cell[2])]
+  if isempty(gi)
```

Comment:
Out of curiosity. Do ideals with zero generators behave badly?

Reply:
The constructor is missing.